AnnoyancesFilter/sections/tweaks.txt

```diff
@@ -1885,7 +1885,12 @@ legionscans.com,a2zapk.com,flinsetyadi.com,theaircurrent.com,fully-fundedscholar
 now.rememberapp.co.kr,melodelaa.link,asus-zenfone.com,techieway.blogspot.com,chronologia.pl#%#//scriptlet("abort-current-inline-script", "document.onselectstart")
 ! #%#//scriptlet("abort-current-inline-script", "$", "contextmenu")
 hinfomax.co.kr,britg.kr,japanxxxmovie.com,sexpox.com,47news.jp,learn-from-homey.blogspot.com,belajarbro.id#%#//scriptlet("abort-current-inline-script", "$", "contextmenu")
+! #%#//scriptlet("set-constant", "disable_copy", "noopFunc")
+jootc.com,dora-guide.com,codedosa.com,gradium.co.kr,semesters.in#%#//scriptlet("set-constant", "disable_copy", "noopFunc")
+! #%#//scriptlet("set-constant", "disable_copy_ie", "noopFunc")
+jootc.com,dora-guide.com,codedosa.com,gradium.co.kr,semesters.in#%#//scriptlet("set-constant", "disable_copy_ie", "noopFunc")
```

Comment:
Better change existing rules
```
dora-guide.com,codedosa.com,gradium.co.kr,semesters.in#%#//scriptlet('set-constant', 'disable_copy', 'noopFunc')
dora-guide.com,codedosa.com,gradium.co.kr,semesters.in#%#//scriptlet('set-constant', 'disable_copy_ie', 'noopFunc')
```

Reply:
Also I see another rules for these domains in this file. Can be removed?

Reply:
Checked.